Client/mods/deathmatch/logic/CClientVehicle.cpp

```diff
@@ -3883,11 +3900,15 @@ void CClientVehicle::SetHeadLightColor(const SColor color)
 //
 
 #if OCCUPY_DEBUG_INFO
-    #define INFO(x)    g_pCore->GetConsole ()->Printf x
-    #define WARN(x)    g_pCore->GetConsole ()->Printf x
+    #define INFO(x) g_pCore->GetConsole()->Printf x
+    #define WARN(x) g_pCore->GetConsole()->Printf x
 #else
-    #define INFO(x)    {}
-    #define WARN(x)    {}
+    #define INFO(x) \
+        { \
+        }
+    #define WARN(x) \
+        { \
+        }
```

Comment:
```suggestion
    #define INFO(x)    {}
    #define WARN(x)    {}
```
Previous code is more readable.